### ISSUE TITLE
Add claim delimiter configuration example in OAuth2 Resource Server JWT documentation

### ISSUE THREAD

**Issue by PatrickWalter214** [spring-projects/spring-security#14373]
**Expected Behavior**

An example for configuring the claim delimiter is listed in the [Extracting Authorities](https://docs.spring.io/spring-security/reference/servlet/oauth2/resource-server/jwt.html#oauth2resourceserver-jwt-authorization-extraction) section of the OAuth2 Resource Server JWT documentation.

**Current Behavior**

The JwtGrantedAuthoritiesConverter has 3 available methods for configuration. But only 2, namely the configuration of the authorities claim and the authorities prefix, are outlined in the documentation.

**Context**

About a year ago I opened [a PR](https://github.com/spring-projects/spring-security/pull/12073) that added the possibility to configure the claim delimiter. Now I would like to outline this configuration option in the Spring documentation. The configuration example I want to add would be the same as in the test of the method, with the relevant lines being:
```
JwtGrantedAuthoritiesConverter jwtGrantedAuthoritiesConverter = new JwtGrantedAuthoritiesConverter();
jwtGrantedAuthoritiesConverter.setAuthoritiesClaimDelimiter(",");
```
With this issue I would like to discuss if it is ok to outline this configuration in the documentation.

**Comment by jzheaux** [comment]
@PatrickWalter214, yes, I think that would be a good addition to the documentation. Would you please add for [servlet](https://github.com/spring-projects/spring-security/blob/main/docs/modules/ROOT/pages/servlet/oauth2/resource-server/jwt.adoc?plain=1#L950) as well as [reactive](https://github.com/spring-projects/spring-security/blob/main/docs/modules/ROOT/pages/reactive/oauth2/resource-server/jwt.adoc?plain=1#L748)?
